docs/included.md

```diff
@@ -117,6 +117,26 @@
   - [API](api.md#scores.continuous.rmse)
   - [Tutorial](project:./tutorials/Root_Mean_Squared_Error.md)
   - [Wikipedia](https://en.wikipedia.org/wiki/Root-mean-square_deviation)
+* - Threshold Weighted Absolute Error
```

Comment:
The docs build process is giving me an error. It says

docs/included.md:5: ERROR: Error parsing content block for the "list-table" directive: two-level bullet list expected, but row 25 does not contain a second-level bullet list

Reply:
I think I may have found the errors in the list table.

It looks like there is an indentation in front of the asterisk for each of:
- Threshold Weighted Expectile Score
- Threshold Weighted Huber Loss
- Threshold Weighted Squared Error
- Threshold Weighted Quantile Score

Threshold Weighted Absolute Error looked okay to me

Reply:
Thanks for this pick up - hopefully I have fixed it